internal/pkg/describe/env.go

```diff
@@ -317,6 +317,39 @@ func (d *EnvDescriber) filterDeployedJobs() ([]*config.Workload, error) {
 	return deployedJobs, nil
 }
 
+// ValidateCFServiceDomainAliases returns error if an environment using cdn is deployed without specifying http.alias for all services
+func (d *EnvDescriber) ValidateCFServiceDomainAliases() error {
+
+	stackDescr, err := d.cfn.Describe()
+	if err != nil {
+		return fmt.Errorf("describe stack: %w", err)
+	}
+
+	servicesString, ok := stackDescr.Parameters[cfnstack.EnvParamALBWorkloadsKey]
+	if !ok {
+		return nil
+	}
+	services := strings.Split(servicesString, ",")
+
+	jsonOutput, ok := stackDescr.Parameters[cfnstack.EnvParamAliasesKey]
+	if !ok {
+		return fmt.Errorf("cannot find %s in env stack parameter set", cfnstack.EnvParamAliasesKey)
+	}
+
+	var aliases map[string][]string
+	err = json.Unmarshal([]byte(jsonOutput), &aliases)
+	if err != nil {
+		return fmt.Errorf("unmarshal \"%s\": %w", jsonOutput, err)
+	}
+	for _, service := range services {
+		if _, ok := aliases[service]; !ok {
+			return fmt.Errorf("all lb web services deployed in an environment with CloudFront enabled must have http.alias specified")
```

Comment:
Maybe we can give a more helpful error message telling users which ones don't have `http.alias` specified?